### PR TITLE
Closing dropdown on click outside of dropdown

### PR DESCRIPTION
**Pull request by emilyzhong**
## Dropdown menus should close when clicking outside
https://app.asana.com/0/860297512787618/1118971998744517/f

### Related PRs
None

### Migrations
None

### Tests Performed, Edge Cases
Clickin'

### Screenshots
click

CC: @by-co

### CODE REVIEW

File: app/assets/javascripts/request_dropdown_toggle.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+document.addEventListener("click", (e) => {
```
Comment (by-co) [inline app/assets/javascripts/request_dropdown_toggle.js]: Does this affect the entire webpage? Is there a way we could have it only be on one specific div :o

Comment (emilyzhong) [reply app/assets/javascripts/request_dropdown_toggle.js]: it affects the entire webpage — i tried having it localized to one div but there isn't any that spans the entire page / even if I add one in, there are issues with detecting the click due to different z-indices and whatnot
